pkg/cmd/roachtest/tests/perturbation/intents.go

```diff
@@ -33,7 +33,22 @@ func (i intents) setup() variations {
 }
 
 func (i intents) setupMetamorphic(rng *rand.Rand) variations {
-	return i.setup().randomize(rng)
+	v := i.setup()
+	v = v.randomize(rng)
+
+	// TODO(139187): Large block sizes result in a big slowdown when the intents
+	// are written.
+	// TODO(139188): Large block sizes result in a big slowdown when the intents
+	// are resolved.
+	if v.blockSize > 1024 {
```

Comment:
optional: `v.blockSize = min(v.blockSize, 1024)` for brevity
Same below.

Reply:
Good idea - I'm going to switch to this style for all of these as it is probably easier to read.